### PR TITLE
drainer/: Add ignore-txn-commit-ts to skip some txn

### PR DESCRIPTION
**Pull request by july2993**
### What problem does this PR solve? 
add ignore-txn-commit-ts to skip some txn.
use case: for some incompatible DDL, user may handle it manaually and want to skip this binlog, currentlly, usert can only change the checkpoint to skip this,  the process is tedious and prone to error.

### What is changed and how it works?
add `ignore-txn-commit-ts` support to skip specified commit ts txn

### Check List 

Tests 

 - Unit test
 - Integration test
 - Manual test (add detailed scripts or steps below)
config `ignore-txn-commit-ts` and make sure it skip it

Code changes

Side effects

Related changes

 - Need to cherry-pick to the release branch
 - Need to update the documentation

### CODE REVIEW

File: drainer/syncer.go [hunk 1]
```diff
@@ -394,6 +399,10 @@ ForLoop:
 				beginTime := time.Now()
 				s.itemsWg.Add(1)
 				lastAddComitTS = binlog.GetCommitTs()
+
+				log.Info("add ddl item to syncer, you can add this commit ts to `ignore-txn-commit-ts` for skip this ddl if need",
```
Comment (suzaku) [inline drainer/syncer.go]: Maybe only output this line when `Sync` failed in `if err != nil`?

Comment (july2993) [reply drainer/syncer.go]: it's ok to log info for every ddl binlog statement(this is important and will not too much).
but it better only log the part `you can add this commit ts to ignore-txn-commit-ts..` only when failing to exec the ddl, but the `Sync` is async, it returns success only means add to syncer success `but not` successfully synced to downstream, so log it for every ddl now.

Comment (kennytm) [inline drainer/syncer.go]:
```suggestion
				log.Info("add ddl item to syncer, you can add this commit ts to `ignore-txn-commit-ts` to skip this ddl if needed",
```

**Pull request merged**